contracts/connection-router/tests/test.rs

```diff
@@ -62,18 +57,26 @@ fn make_chain(name: &str, config: &mut TestConfig) -> Chain {
 }
 
 fn register_chain(config: &mut TestConfig, chain: &Chain) {
-    let _ = config
-        .app
-        .execute_contract(
-            config.governance_address.clone(),
-            config.contract_address.clone(),
-            &ExecuteMsg::RegisterChain {
-                chain: chain.chain_name.clone(),
-                gateway_address: chain.gateway.to_string(),
-            },
-            &[],
-        )
-        .unwrap();
+    let _ = config.connection_router.execute(
+        &mut config.app,
+        config.governance_address.clone(),
+        &ExecuteMsg::RegisterChain {
+            chain: chain.chain_name.clone(),
+            gateway_address: chain.gateway.to_string(),
+        },
+    );
```

Comment:
be careful not to throw `Result<>` away. we might not be interested in the response value, but keep the `unwrap` so the tests fails if the registration fails

Reply:
`fn register_chain(config: &mut TestConfig, chain: &Chain) {
    config.connection_router.execute(
        &mut config.app,
        config.governance_address.clone(),
        &ExecuteMsg::RegisterChain {
            chain: chain.chain_name.clone(),
            gateway_address: chain.gateway.to_string(),
        },
    ).unwrap();
}`

Reply:
it's still good to keep the `let _ =`, so it is obvious that it's discarded on purpose and not just forgotten